index.html

```diff
@@ -40,6 +40,10 @@ <h1 class="section-title">Join us!</h1>
         We meet the last Tuesday of every month, 6:30 PM. Join our Discord here:
       </p>
       <a href="https://discord.gg/fqUMmXPU5c">https://discord.gg/fqUMmXPU5c</a>
+      <p>
+        Visit my GitHub repository here:
```

Comment:
How about we say `our` GitHub repo instead :) 
Feel free to squash after making the change (e.g. `git rebase -i HEAD~2` or a `git reset --soft HEAD~2` and so on :)